cli/CreateTestCasesCommand.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Yaml\Yaml;
+use UAParser\Parser;
+
+class CreateTestCasesCommand extends Command
+{
+    /**
+     * @var array $sources ua-parser/uap-core test-case data sources
+     */
+    protected $sources = [
+        [
+            'name' => 'device',
+            'key'  => 'uc-d',
+            'url'  => 'https://raw.githubusercontent.com/ua-parser/uap-core/master/tests/test_device.yaml'
+        ],
+        [
+            'name' => 'operating system',
+            'key'  => 'uc-o',
+            'url'  => 'https://raw.githubusercontent.com/ua-parser/uap-core/master/tests/test_os.yaml'
+        ],
+        [
+            'name' => 'user agent',
+            'key'  => 'uc-u',
+            'url'  => 'https://raw.githubusercontent.com/ua-parser/uap-core/master/tests/test_ua.yaml'
+        ]
+    ];
+
+    /**
+     * @var array $testCaseData Array to append test case data to before parsing to YAML
+     */
+    protected $testCaseData = [];
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure(): void
+    {
+        $this->setName('create-test-cases')
+             ->setDescription('Create test-case data for unit tests: `tests/test-cases.yaml`');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $outputFile = dirname(__DIR__) . '/tests/test-cases.yaml';
+        if (!is_file($outputFile)) {
+            throw new \RuntimeException("Output file does not exist: {$outputFile}");
+        }
+        if (!is_writable($outputFile)) {
+            throw new \RuntimeException("Output file is not writable: {$outputFile}");
+        }
+        if (!is_readable($outputFile)) {
+            throw new \RuntimeException("Output file is not readable: {$outputFile}");
+        }
+
+        $io = new SymfonyStyle($input, $output);
+        $io->title('Loading Existing Test Cases');
+
+        $io->text("Parsing test cases from: <comment>{$outputFile}</comment>");
+        $this->testCaseData = Yaml::parse(file_get_contents($outputFile));
+
+        $totalRecords = count($this->testCaseData);
+        $io->text("<info>Parsed {$totalRecords} records</info>");
+
+        $io->title('Fetching New Test Cases');
```

Comment:
SymfonyStyle suggests that `title()` should only be used once to show the command's name (or using at the start of the process, like the classify command seems good too).

Perhaps these need to be `section()` instead?